ods_ci/tests/Resources/OCP.resource

```diff
@@ -71,6 +71,14 @@ Is RHODS Self-Managed
     ${rc}=    Run And Return Rc    oc get catalogsource addon-managed-odh-catalog -n ${OPERATOR_NAMESPACE}
     Run Keyword And Return    Run Keyword And Return Status      Should Not Be Equal As Numbers    ${rc}    ${0}
 
+Is Disconnected Cluster
```

Comment:
Hi @jgarciao I would rather use a different terminology, for example `Is PIP Mirror Available` from 2 reasons
1. Testing with PIP miror is possible also in an online env
2. Deciding if a cluster is disconnected just according to presence of PIP vars is risky IMHO

Reply:
I've refactored the code to just talk about pip_index_url, which will be used for connected and disconnected clusters